src/data/reducers/assets.js

```diff
@@ -181,6 +184,17 @@ export const status = (state = {}, action) => {
   }
 };
 
+export const imagePreview = (state = imagePreviewInitial, action) => {
```

Comment:
Because I made the filter a separate element, it is rendered by the AssetsPage. The state that the AssetsImagePreviewFilter component is responsible for is also needed by the AssetsTable in order to show/hide the image preview column. In order for them to both have that piece of state, I could either lift the state up into the AssetsPage or move it into Redux. I decided to go with Redux, since I thought it would be a little more intuitive, but we could just lift the state up. AssetsPage could pass a callback into the AssetsImagePreviewFilter and then pass a isImagePreviewEnabled prop to the AssetsTable. Any opinions?